snippets/logging-retention-period-snip.adoc

```diff
@@ -10,5 +10,5 @@
 
 [IMPORTANT]
 ====
-If there is no retention period defined on the s3 bucket or in the LokiStack custom resource (CR), then the logs are not pruned and they stay in the s3 bucket forever, which might fill up the s3 storage.
-====
\ No newline at end of file
+If there is no retention period defined on the S3 bucket or in the LokiStack custom resource (CR), then the logs are not pruned and they stay in the S3 bucket forever, which might fill up the S3 storage.
```

Comment:
How about replace s3 with object storage buckets?   as  the bucket may not s3. it can be swift, gcs, Azure Blog too.